GooglePlayInstant/Editor/PlayInstantBuilder.cs

```diff
@@ -29,6 +31,102 @@ public static class PlayInstantBuilder
     {
         private const string BuildErrorTitle = "Build Error";
 
+        /// <summary>
+        /// Displays an error or warning dialog if there is an issue that would prevent a build from succeeding.
+        /// This method performs the checks from <see cref="CheckBuildPrerequisites"/>, but also performs checks
+        /// relevant to builds intended for publishing on Play Console.
+        /// </summary>
+        /// <returns>True if all prerequisites are met, or false if one failed.</returns>
+        public static bool CheckBuildAndPublishPrerequisites()
+        {
+            if (!CheckBuildPrerequisites())
+            {
+                return false;
+            }
+
+            // This is a warning that can be skipped, so it should be checked last.
+            // BuildTargetGroup.Unknown is used when checking the project's "Default Icon".
+            if (!HasIconForTargetGroup(BuildTargetGroup.Unknown) && !HasIconForTargetGroup(BuildTargetGroup.Android))
```

Comment:
Does it make more sense to throw this into PlayInstantSettingsPolicy.cs as a recommended policy?

Reply:
That is a potential alternative. However, based on the discussion in the bug, etc, I thought it made the most sense to put it here because:
- when someone is first experimenting with Build and Run they shouldn't have to have to worry about icons
- the existing policy settings are all cases where with a single "Update" button click the user can switch the setting; creating and configuring an icon doesn't easily fit that model
- when someone has gotten far enough along to be thinking about publishing, that's when I want them to think about icons
- assuming you believe the previous line of reasoning, if this were in the policy settings we'd have to make policy settings context-aware (i.e. app is ready to publish), and we'd have to make this setting required (otherwise it wouldn't prompt and block the build)

Reply:
That all makes sense. LGTM then.